keychain.h

```diff
@@ -0,0 +1,70 @@
+#include <assert.h>
```

Comment:
meh :/

Reply:
Neither am I ;) However, if the result is a Password where either an Error or Success was expected, it means that I made a mistake in the implementation, not that the user's input (or any magic state in the OS) lead to an error.

Reply:
True, but it's still a runtime check, which could (obviously shouldn't but _could_) happen in an enduser system, killing it even though, potentially, the rest of the program is still working correctly.

Reply:
However using an `error_code` instead of a `std::variant` would make this obsolete anyway :)